runtime/darwinia/src/pallets/bridge_dispatch.rs

```diff
@@ -126,3 +128,15 @@ impl Config<WithCrabDispatch> for Runtime {
 	type TargetChainAccountPublic = bp_darwinia::AccountPublic;
 	type TargetChainSignature = bp_darwinia::Signature;
 }
+impl Config<WithDarwiniaParachainDispatch> for Runtime {
+	type AccountIdConverter = bp_darwinia_parachain::AccountIdConverter;
+	type BridgeMessageId = (LaneId, MessageNonce);
+	type Call = Call;
+	type CallValidator = CallValidator;
+	type EncodedCall = bm_darwinia_parachain::FromDarwiniaParachainEncodedCall;
+	type Event = Event;
+	type IntoDispatchOrigin = IntoDispatchOrigin;
+	type SourceChainAccountId = bp_darwinia_parachain::AccountId;
+	type TargetChainAccountPublic = bp_darwinia::AccountPublic;
+	type TargetChainSignature = bp_crab::Signature;
```

Comment:
This should be `type TargetChainSignature = bp_darwinia::Signature;`. This dispatch module works on the darwinia side. The target chain is `darwinia`, source chain refers to `darwinia-parachain`.